dotnet/test/common/TargetLocatorTest.cs

```diff
@@ -117,29 +118,22 @@ public void ShouldSwitchToFrameByIndexAndBackToDefaultContent()
             Assert.AreEqual(driver.FindElement(By.Id("pageNumber")).Text, "1");
 
             driver.SwitchTo().DefaultContent();
-            try
-            {
-                // DefaultContent should not have the element in it.
-                Assert.AreEqual(driver.FindElement(By.Id("pageNumber")).Text, "1");
-                Assert.Fail("Should not be able to get element in frame from DefaultContent");
-            }
-            catch (NoSuchElementException)
-            {
-            }
+
+            // DefaultContent should not have the element in it.
+            Assert.That(
+                () => Assert.AreEqual(driver.FindElement(By.Id("pageNumber")).Text, "1"),
```

Comment:
Assert that Assert is strange.

Reply:
Ah yeah my bad, I just copied everything in the `try` block.

The whole thing is unnecessary, `driver.FindElement` is expected to throw.